src/utils/log.c

```diff
@@ -140,7 +140,11 @@ void log_msg(log_level level, const char *fmt, ...) {
         va_start(args, fmt);
         vfprintf(target->fp, fmt, args);
         va_end(args);
-        fprintf(target->fp, "\n");
+        if(state->colors && target->colors) {
```

Comment:
Unrelated change?

Reply:
Yeah, this unrelated change just clears the colors after when we call log_debug with a color.
It's in a seperate commit so things don't get confused